presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -731,8 +731,10 @@ exists as we've rolled back to the previous state.
     -----------+------+-----------+---------
     (0 rows)
 
-Iceberg Connector Limitations
------------------------------
+Table with deleted files
```

Comment:
> deleted files

This sounds like files have been deleted... so it is strange that they can be read. I think you want to refer to Delete files, not deleted files. 

CC: @steveburnett

Reply:
Thanks for this, @mbasmanova! I have reread the spec, the draft, and your comment, and I will follow up with a new review based on this. I appreciate your helping me understand this better.